tests/structured_tests/split_7.toml

```diff
@@ -0,0 +1,11 @@
+query = "* | logfmt | split with \" \""
```

Comment:
queries with quotes are a little easier to read in `"""`

Reply:
Good point - done for all tests!